pom.xml

```diff
@@ -150,7 +150,9 @@
         <cxf.version>3.2.6</cxf.version>
         <ftpserver-version>1.0.6</ftpserver-version>
         <geotools.version>19.1</geotools.version>
-        <guava.version>20.0</guava.version>
+        <guava.version-number>25.1</guava.version-number>
+        <guava.platform>jre</guava.platform>
+        <guava.version>${guava.version-number}-${guava.platform}</guava.version>
```

Comment:
✏️ I don't think splitting out the `version-number` and `platform` is necessary

Reply:
I had to. Package versions do not like the `-` symbol. See the OpenSAML pom. 
https://github.com/codice/ddf/pull/5379/files#diff-22809f9791373339f3187a25d0ccc590R32